Wikipedia/Code/WMFCompassView.m

```diff
@@ -223,4 +223,21 @@ - (void)drawCompassLinesInContext:(CGContextRef)ctx
     }
 }
 
+#pragma mark - Accessibility
+
+- (BOOL)isAccessibilityElement {
+    return YES;
+}
+
+- (NSString *)accessibilityLabel {
+    double positive_angle = self.angleRadians >= 0 ? self.angleRadians : self.angleRadians + 2*3.14;
+    NSInteger clock_direction = lroundf(positive_angle / (3.14 / 6));
```

Comment:
would you mind writing unit tests for these calculations? e.g. that some example angles return the expected "clock" orientation